unicoremc/models.py

```diff
@@ -301,6 +309,23 @@ def get_website_manager(self):
     def frontend_url(self):
         return 'http://%s' % self.get_generic_domain()
 
+    def apollo_frontend_url(self):
+        hub = 'qa-apollo' if settings.DEPLOY_ENVIRONMENT == 'qa' else 'apollo'
+        return "%(country)s-%(app_type)s.%(hub)s.unicore.io" % {
+            'country': self.country.lower(),
+            'app_type': self.app_type,
+            'hub': hub
+        }
```

Comment:
I'm a little bit confused as we need functions with a temporary name in them? Is there a way we can make this things config variables somehow?